test_settings.py.sample

```diff
@@ -1,2 +1,7 @@
-USERNAME = 'USERNAME'
-PRIVATE_KEY = 'PRIVATE_KEY'
+CONNECTION_OPTIONS = {
```

Comment:
Is it really sample? Or complete settings without user/key pair? If so, maybe it makes sense to ask user to just update that file with user/key info before running tests?